tests/cli/commands/test_dag_command.py

```diff
@@ -139,6 +139,12 @@ def test_show_dag_print(self):
         self.assertIn("graph [label=example_bash_operator labelloc=t rankdir=LR]", out)
         self.assertIn("runme_2 -> run_after_loop", out)
 
+    def test_generate_dag_yaml(self):
+        with contextlib.redirect_stdout(io.StringIO()) as temp_stdout:
+            dag_command.generate_pod_yaml(self.parser.parse_args([
+                'dags', 'generate_yaml', '--dag-id', 'example_bash_operator']))
+
```

Comment:
Can you add even one assertion?

Reply:
@mik-laj yes I plan to add assertions that a yaml file is created and that it's a valid pod.

Reply:
@mik-laj PTAL added assertions